ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.mm

```diff
@@ -31,9 +31,53 @@
 }
 #pragma clang diagnostic pop
 
-void CKDetectComponentScopeCollisions(const CKComponentLayout &layout)
+static CKComponent *completeComponentScopeCollisionPair(CKComponent *collidingComponent,
+                                            id<NSObject> collidingScope,
+                                            NSMapTable<CKComponent *, CKComponent*> *componentsToParentComponents)
+{
+  for (CKComponent* componentKey in componentsToParentComponents) {
```

Comment:
Not trying to start a war of the kind of Vim vs Emacs
Just my two cents.
I know It's very common to have the type and then a space and the `Type *variable`
However I don't understand that. 
For me, makes more sense `Type* variable` Because I think the `*` belong to the type not the variable name. Does that make sense?